hack/lima-kubestellar.yaml

```diff
@@ -0,0 +1,179 @@
+    case "$KIND_ARCH" in
+      x86_64) KIND_ARCH="amd64" ;;
+      aarch64) KIND_ARCH="arm64" ;;
+    esac
+
+    curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.20.0/kind-linux-${KIND_ARCH}
+    chmod +x ./kind
+    sudo mv ./kind /usr/local/bin/kind
+
+    curl -L https://raw.githubusercontent.com/open-cluster-management-io/clusteradm/main/install.sh | bash
+
+    wget -q https://raw.githubusercontent.com/kubestellar/kubestellar/refs/heads/main/scripts/create-kubestellar-demo-env.sh
+    chmod +x create-kubestellar-demo-env.sh
+    ./create-kubestellar-demo-env.sh --platform kind
+  
+probes:
+- description: "wait for KubeFlex to be installed"
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+    if ! timeout 30s bash -c "until command -v kflex >/dev/null 2>&1; do sleep 3; done"; then
+      echo >&2 "KubeFlex is not installed."
+      exit 1
+    fi
+- description: "wait for KubeFlex to be available"
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+    if ! timeout 30s bash -c "until kubectl config use-context kind-kubeflex >/dev/null 2>&1; do sleep 3; done"; then
+      echo >&2 "KubeFlex is not available."
+      exit 1
+    fi
+- description: "wait for KubeStellar Core to be available"
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+
+    if ! timeout 300s bash -c "until kflex ctx --overwrite-existing-context its1 >/dev/null 2>&1; do sleep 3; done"; then
+      echo >&2 "KubeStellar Core is not available."
+      exit 1
+    fi
+- description: "wait for managedcluster cluster1 to be available"
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+
+    if ! timeout 300s bash -c "until kubectl --context its1 get managedclusters | grep -q cluster1 >/dev/null 2>&1; do sleep 3; done"; then
+      echo >&2 "managedcluster cluster1 is not available."
+      exit 1
+    fi
+
+- description: "wait for managedcluster cluster2 to be available"
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+
+    if ! timeout 300s bash -c "until kubectl --context its1 get managedclusters | grep -q cluster2 >/dev/null 2>&1; do sleep 3; done"; then
+      echo >&2 "managedcluster cluster2 is not available."
+      exit 1
+    fi
+
+  hint: Use "sudo tail -f /var/log/cloud-init-output.log" in the guest to see progress.
+hostResolver:
+  # hostResolver.hosts requires lima 0.8.3 or later. Names defined here will also
+  # resolve inside containers, and not just inside the VM itself.
+  hosts:
+    host.docker.internal: host.lima.internal
+portForwards:
+- guestSocket: "/run/user/{{.UID}}/docker.sock"
+  hostSocket: "{{.Dir}}/sock/docker.sock"
+message: |
+  ------
+  host_context=kind-kubeflex
+  its_cp=its1
+  its_context=its1
+  wds_cp=wds1
+  wds_context=wds1
+  wec1_name=cluster1
+  wec2_name=cluster2
+  wec1_context=$wec1_name
+  wec2_context=$wec2_name
+  label_query_both=location-group=edge
+  label_query_one=name=cluster1
+
+  To access the KubeStellar demo environment from the host, run the following commands:
```

Comment:
"host" is not entirely clear here, since there are multiple levels of virtualization involved here. In particular, I think that this text is NOT trying to refer to the host where the `limactl` command is run, but rather is trying to talk about the guest VM that `limactl start` creates --- which is, in turn, the host of the `kind` clusters.